src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -1,46 +1,34 @@
-import { bchChainId, CHAIN_NAMESPACE, dogeChainId, ethChainId } from '@shapeshiftoss/caip'
+import { btcChainId, dogeChainId, ethChainId } from '@shapeshiftoss/caip'
 import { describe, expect, it } from 'vitest'
 
+import { UTXO_MAXIMUM_BYTES_LENGTH } from '../constants'
```

Comment:
@NeOMakinG do we want to make this an exhaustive mapping by ChainId? i.e `getMaxBytesLengthByChainId` with an exhaustive switch, returning 80 for BTC and its forks, 220 for BCH weird non-standard fork (a.k.a BCH) and `Infinity` for others?

Alternatively, I would use `BTC_MAXIMUM_BYTES_LENGTH` instead of BCH - my main issue here is the naming implies to all UTXOs, whereas  BTC makes it clear this is for BTC only (and DOGE/LTC just happen to share the same params as BTC as they're pretty identical forks)

Reply:
I did a util `getMaxBytesLengthByChainId` which does the mapping as you said, but not that exhaustive actually because It would be a non-sense for now as it would double the work in case we need to add another UTXO chain, we already have an exhaustive list of these chains somewhere else